testprojects/tests/java/org/pantsbuild/testproject/workdirs/onedir/WorkdirTest.java

```diff
@@ -0,0 +1,18 @@
+// Copyright 2015 Pants project contributors (see CONTRIBUTORS.md).
+// Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+package org.pantsbuild.testproject.workdirs.onedir;
+
+import org.junit.Test;
+import java.io.File;
+
+import static org.junit.Assert.assertTrue;
+
+/** Ensure test_cwd works correctly. */
+public class WorkdirTest {
+    @Test
+    public void testPlaceholderExists() {
+        assertTrue("Could not find placeholder.txt, working directory must be wrong!",
+                   new File("placeholder.txt").exists());
```

Comment:
Add some contents to this file "this goes along with testprojects/tests/java/org/pantsbuild/testproject/workdirs/onedir/WorkdirTest.java" so that someone doesn't accidentally delete it or move it.